src/video/video.manager.ts

```diff
@@ -64,6 +64,7 @@ export class VideoManager implements VideoRepository {
 
     static async getMany(userId: string, videoFilter: Partial<IVideo>) {
         const userClassifications = await UserClassificationManager.getUserClassifications(userId);
+        if(userId != videoFilter.owner) videoFilter = {...videoFilter, published: true, status: VideoStatus.READY};
```

Comment:
Don't forget to use linter... your code is not aligned well... Realign and refactor

Reply:
fixed